NAMESPACE

```diff
@@ -22,10 +23,12 @@ importFrom(htmltools,tagList)
 importFrom(htmltools,tags)
 importFrom(jsonlite,toJSON)
 importFrom(shiny,NS)
+importFrom(shiny,addResourcePath)
 importFrom(shiny,getDefaultReactiveDomain)
 importFrom(shiny,icon)
 importFrom(shiny,is.reactive)
 importFrom(shiny,isTruthy)
 importFrom(shiny,reactive)
 importFrom(shiny,renderText)
+importFrom(shiny,tags)
```

Comment:
Let's always import tags from htmltools.  htmltools is the package that originally exports this object and shiny just reexports it.

Reply:
Good to know, thx.